packages/js/src/ui/components/Notification/DefaultNotification.tsx

```diff
@@ -21,6 +21,20 @@ export const DefaultNotification = (props: DefaultNotificationProps) => {
   const style = useStyle();
   const { t, locale } = useLocalization();
   const { status } = useInboxContext();
+  const [minutesPassed, setMinutesPassed] = createSignal(0);
+  const date = createMemo(() => {
+    minutesPassed(); //register as dep
+
+    return formatToRelativeTime({ fromDate: new Date(props.notification.createdAt), locale: locale() });
+  });
+
+  createEffect(() => {
+    const interval = setInterval(() => {
+      setMinutesPassed((prev) => prev + 1);
+    }, 1000);
```

Comment:
❓ **question:** WDYT about making it smart and calculating for example the time passed:
- if it's < 60s then update it every second
- if it's >= 1m but < 60m update it every minute
- if it's >= 1h but < 24h update it every hour
Like to control the interval as well so we don't update the DOM unnecessarily, it shouldn't be hard.

Reply:
A recalculation of the date every minute is nothing performance wise

Reply:
We can just do it every second, but I did once per minute as that's what we decided. Also, showing the seconds instead of just now will be spammy for the first minute.

Reply:
I see the interval is set for every second.